extensions/azurehybridtoolkit/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "azurehybridtoolkit",
+  "displayName": "%displayName%",
+  "description": "%description%",
+  "version": "0.1.0",
+  "publisher": "Microsoft",
+  "engines": {
+    "vscode": "*",
+    "azdata": "*"
+  },
+  "main": "./out/main",
+  "activationEvents": [
+    "*"
+  ],
+  "contributes": {
+    "commands": [
+      {
+        "command": "hybridtoolkit.openNotebooks",
+        "title": "%title.openNotebookFolder%",
+        "icon": {
+          "dark": "resources/dark/placeholder_logo_jupyterhub.svg",
+          "light": "resources/light/placeholder_logo_jupyterhub.svg"
+        }
+      }
+    ],
+    "menus": {
+      "commandPalette": [
+        {
+          "command": "hybridtoolkit.openNotebooks",
```

Comment:
You can just remove the entire `commandPalette` entry - commands are automatically included in the palette by default.